x-pack/plugins/security_solution/public/detections/components/take_action_dropdown/index.tsx

```diff
@@ -182,22 +170,15 @@ export const TakeActionDropdown = React.memo(
       onInvestigateInTimelineAlertClick: closePopoverHandler,
     });
 
-    const osqueryAvailable = osquery?.isOsqueryAvailable({
```

Comment:
This one on the contrary I am sure happens only once - because there is a single flyout opened.